templates/layouts/default.pug

```diff
@@ -45,6 +45,8 @@ html
                 i.icon.fab.fa-discord
               li: a.icon(href="https://www.youtube.com/c/ForgedAllianceForever")
                 i.fab.fa-youtube
+              li: a.icon(href="https://www.twitch.tv/faflive")
```

Comment:
faflive is offline 99% of the time. Wouldn't it be better to link to the FA category of twitch?